frontend/src/index.jade

```diff
@@ -90,7 +90,8 @@ html
             .summary-chart(v-for="(user, i) in repo")
               .summary-chart__title(
                 title="click to view author's code",
-                v-on:click="$emit('view-authorship', {author:user.name, repo:user.repoName, name:user.displayName})"
+                v-on:click="$emit('view-authorship', "
+                    + "{author:user.name, repo:user.repoName, name:user.displayName, totalCommits:user.totalCommits})"
```

Comment:
why is this change necessary?

Reply:
@eugenepeh this change ensures that the button is not displayed when there are no files i.e. total contribution = 0.

Reply:
do it like this instead:

```
.toolbar(v-if="tabInfo.tabAuthorship.totalCommits > 0")
    a(v-on:click="isCollapsed=false", onclick="expandAll(true)") Expand all
    a(v-on:click="isCollapsed=true", onclick="expandAll(false)") Collapse all
```